packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -71,29 +132,29 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
         }
         deltaX *= 10;
       }
+      props.onMove?.(e);
       // if moving up/down only, no need to resize
       if (deltaX !== 0) {
         columnResizeWidthRef.current += deltaX;
-        stateRef.current.onColumnResize(item, columnResizeWidthRef.current);
-        props.onMove(e);
+        resize(item, columnResizeWidthRef.current);
       }
     },
     onMoveEnd(e) {
       let {pointerType} = e;
       columnResizeWidthRef.current = 0;
-      props.onMoveEnd(e);
+      props.onMoveEnd?.(e);
       if (pointerType === 'mouse') {
-        stateRef.current.onColumnResizeEnd(item);
+        endResize(item);
       }
     }
   });
+
   let min = Math.floor(stateRef.current.getColumnMinWidth(item.key));
   let max = Math.floor(stateRef.current.getColumnMaxWidth(item.key));
   if (max === Infinity) {
     max = Number.MAX_SAFE_INTEGER;
   }
   let value = Math.floor(stateRef.current.getColumnWidth(item.key));
```

Comment:
@snowystinger With keyboard interaction and focus on the visually hidden `input`, the `value` should update with each ArrowLeft/Right key press, so that the `value` and `aria-valuetext` update to announce the new width. It seem that at present, the slider value only gets updated after the move ends.

Reply:
Hmmm I definitely see it being updated on every arrow key, as well as live when dragging. But I'm not hearing it change with VO.
I tried out slider and it also doesn't announce the new values. So I think this is fine if slider is fine?